commands/generate-key.js

```diff
@@ -1,45 +1,50 @@
 const KeyPair = require('near-api-js').KeyPair;
```

Comment:
changes in this file seem to be unrelated to the topic of PR, best to revert them